Src/MainLib/InterceptFrame.cpp

```diff
@@ -10,6 +10,10 @@
 #include "ImageSaveManager.h"
 #include "MovieMaker.h"
 
+#ifdef GLI_BUILD_WINDOWS
```

Comment:
Is this windows.h include needed here? It should be pulled from a lot of headers that include osdefines.h which has windows.h

Reply:
Hmm, I thought it is, I will double check.

Reply:
Right, it wasn't needed, I removed it.